tests/testthat/test-help.R

```diff
@@ -22,6 +22,12 @@ test_that("shim_help behaves the same as utils::help for nonexistent objects", {
   expect_equal(length(shim_help("foofoo")), 0)
 })
 
+test_that("shim_help works with complex NULL `package = ` argument (#266)", {
+  expect_equal(
+    class(pkgload:::shim_help(list, package = (NULL))),
```

Comment:
not sure if you also want something like this as a test

```r
package <- NULL
pkgload:::shim_help("arg_match0", package = (package))
```

possibly this is just captured by your current test

Reply:
I've added it just in case